### PR TITLE
Markdown fixes and minor edits to PR docs.

### PR DESCRIPTION
**Pull request by SaintPeter**
Some markdown and formatting fixes.  Added more explicit text about using the `staging` branch at a user's request..

### CODE REVIEW

File: How-To-Create-A-Pull-Request-for-Free-Code-Camp.md [hunk 1]
```diff
@@ -12,62 +12,72 @@ There are two methods of creating a Pull for Free Code Camp:
...
-Use a name like `fix/short-fix-description` or `feature/short-feature-description`  
+2. Ensure you are on the `staging` branch using `git status`:
+```
+$ git status
+On branch staging
+Your branch is up-to-date with 'origin/staging'.
+
+nothing to commit, working directory clean
+```
+3. If you are not on staging or your working directory is not clean, resolve any outstanding files/commits and checkout staging
```
Comment (bugron) [inline How-To-Create-A-Pull-Request-for-Free-Code-Camp.md]: From here a new ordered list is started.

Comment (SaintPeter) [inline How-To-Create-A-Pull-Request-for-Free-Code-Camp.md]: Do you know how to fix this?

Comment (bugron) [inline How-To-Create-A-Pull-Request-for-Free-Code-Camp.md]: Put a space at the beginning of lines 22 to 28, worked for me.

File: How-To-Create-A-Pull-Request-for-Free-Code-Camp.md [hunk 2]
```diff
@@ -12,62 +12,72 @@ There are two methods of creating a Pull for Free Code Camp:
 1. Editing files on a local clone
```
Comment (bugron) [inline How-To-Create-A-Pull-Request-for-Free-Code-Camp.md]: 1. -> 2.

**Comment by SaintPeter** [comment]
Updates pushed.

**Comment by bugron** [comment]
Merged, thanks! :+1:

**Pull request merged**